input_cloud: /velodyne_points
min_cluster_size: 3
max_cluster_size: 10
radius_search: 300
cluster_tolerance: 500
min_bbox_size: 0
max_bbox_size: 3